src/components/playlist/playlistFooter.tsx

```
@@ -0,0 +1,7 @@
+const PlaylistFooter = () => {
+    return (
+        <footer className="flex flex-row items-center justify-between w-full mb-[30%] max-sm:mb-[50%]"></footer>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)